src/components/Layout/Section/RSVDetails.tsx

```diff
@@ -22,121 +22,133 @@ const stats = {
 };
 
 type Props = {
-  radschnellweg: {
-    ref: string;
-    name: string;
-    website: string;
-    status: string;
-    to: string;
-    from: string;
-    finished?: string;
-    costs?: number;
-    coordinates: Array<Array<number>>;
+  meta: {
```

Comment:
Eindrücke zu den Types:
- Das JSON unten ist häufig leer `lorem: ""`. Damit ist type string richtig. Aber wollen wir das? Sollte das JSON nicht eher null oder undefined sein, damit wir einfacher über die Daten iterieren können?
- Sind alle attribute Pflicht oder könne einzelne optional sein? Also bspw. `cost?: string`.
- Wenn wir mit den Daten im JS weiter arbeiten, könnte es sich lohnen die Types noch schärfer zu machen, bspw. ```website: `https://${string}` ``` und `state: 'name' | 'name2' | …`
- Die stakeholders fehlen in den Types noch, richtig?

So lange Type-Definitionen empfehle ich in eine `types.ts` mit einem names export wie `RSVDetailsProps` zu verschieben. Das macht diese Datei leichter zu lesen.

Reply:
Habe noch was vergessen, gestern: Wie würde man solche großen JSON Files "absichern" vor unbekannten oder falschen key/values? Ich denke, dafür würde man https://json-schema.org/ nutzen, richtig? Habe aber noch nicht verstanden, wie mir das im Editor hilft. Es scheint dann so etwas wie https://www.npmjs.com/package/json-schema-to-typescript zu geben um aus dem Schema Types zu generieren. Das klingt interessant.

Reply:
Vielleicht macht es Sinn im [Radschnellwege Repo](https://github.com/FixMyBerlin/radschnellwege)  die entsprechenden Types zu hinterlegen. Dann sind sie zentral verfügbar.

Ich hatte mich zuerst an JSON Schemas versucht. Diese sind aber nochmal komplizierter aus meiner Sicht, wie die TypeScript types.

Reply:
Hier ist eine implementierung von einem JSON schema für die verschiedenen geojson typen: https://github.com/geojson/schema . Evtl. ein guter start um von dort aus unsere zusätzlichen metadaten hinzuzufügen.

Reply:
Alright, dann werde ich mich mal daran versuchen ein JSON Schema dafür zu schreiben :)

Reply:
> Habe aber noch nicht verstanden, wie mir das im Editor hilft.

An sich würde das JSON Schema ermöglichen, dass wir das JSON selbst dagegen validieren können. Sehe den Anwendungsfall bei uns aber noch nicht so recht, außer dass wir daraus die Types generieren können.

Reply:
Created first version of MetaJSON Schema: https://github.com/FixMyBerlin/radschnellwege/pull/7